### DISCUSSION TITLE
How to add a submit button?

### DISCUSSION THREAD

**Discussion by RaTeNo** [sa-si-dev/virtual-select#214] [Q&A] [answered]
How to add a submit button?

**Comment by sa-si-dev** [comment]
> How to add a submit button?

@RaTeNo Could you please give some more information about your question?

**Reply by RaTeNo** [reply]
![Bootstrap Select - examples   tutorial - Google Chrome_2022-12-31_13_20_09](https://user-images.githubusercontent.com/6961843/210133126-5f0033af-ae13-4143-abcf-d0cdf091cd61.png)

**Reply by RaTeNo** [reply]
Save button custom

**Comment by sa-si-dev** [comment, marked as answer]
@RaTeNo As of now, it is not possible to a button inside dropbox.

**Comment by exajs161231** [comment]
Is there a way how to retain the selected options upon form submission?

I have created a filter on my website. When I select the options I want to filter and submit the form, the dropdown resets. I want to retain the selected options. Is it possible? Is it the parameter "keepValue"?

**Reply by sa-si-dev** [reply]
> Is there a way how to retain the selected options upon form submission?
> 
> I have created a filter on my website. When I select the options I want to filter and submit the form, the dropdown resets. I want to retain the selected options. Is it possible? Is it the parameter "keepValue"?

@exajs161231 Drop-down value won't be reset on form submit. You can confirm it in the [examples page](https://sa-si-dev.github.io/virtual-select/#/examples?id=validation).
